test/test_corpus.cc

```diff
@@ -132,6 +132,56 @@ unique_ptr<Diagnostic> errorToDiagnostic(const core::GlobalState &gs, const core
     return make_unique<Diagnostic>(Range::fromLoc(gs, error.loc), error.header);
 }
 
+class ErrorHandler {
```

Comment:
Any strong feelings on name? Should this be something like "ExpectationChecker"?

Reply:
I ended up going with the Solomonic choice of `ExpectationHandler`.